### PR TITLE
Added function scrollToLastRow to make cycle cell visible automatical…

### PR DESCRIPTION
**Pull request by benleelawrence**
…ly on the last test plan

### CODE REVIEW

**Review by JasonRitz — APPROVED** [review]
Everything looks and works as it should. Good Job!

File: Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift [hunk 1]
```diff
@@ -165,6 +172,13 @@ extension TestListViewController: EndpointDelegate {
                         self.testCycleList.testCycleList.insert(emptyCycle, at: 0)
                         self.testList.reloadData()
                     }
+                    self.testList.reloadData()
```
Comment (dwagg) [inline Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift]: Is this call needed? If not can we get rid of it. If it is needed can we get rid of the call in the if statement 2 lines up.

File: Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift [hunk 2]
```diff
@@ -165,6 +172,13 @@ extension TestListViewController: EndpointDelegate {
...
+                    
+                    //At the last test plan, auto scroll down to make the cycle cell visible
+                    if self.selectedPlanIndex == self.totalPlansReturnedFromServer - 1 {
+                        self.scrollToLastRow()
+                }
```
Comment (dwagg) [inline Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift]: This curly brace is at the wrong level of indentation.

**Review by dwagg — CHANGES_REQUESTED** [review]
I added a couple of comments to the changes, could you take a look and see if those changes make sense?

**Review by dwagg — APPROVED** [review]

**Pull request merged**